memory/transparent_hugepages_defrag.py

```diff
@@ -121,8 +121,8 @@ def tearDown(self):
         if self.mem_path:
             self.log.info('Cleaning Up!!!')
             memory.set_num_huge_pages(0)
-            process.system('rm -rf %s/*' % self.mem_path, ignore_status=True)
             self.device.unmount()
+            process.system('rm -rf %s' % self.mem_path, ignore_status=True)
```

Comment:
Yes...before the directory contents only were removed and the directory remained as such.
So if other thp tests ran with same job the tests were failing like directory present already
So removed directory too in this PR